src/App.js

```diff
@@ -1,7 +1,53 @@
+import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import ClientList from './components/ClientList';
+
+const getClients = async () => {
```

Comment:
Here my [suggestion](https://github.com/ebauger/plaplante-lab-fun/commit/854557b6e244ee3b216ef4a738faefff0461031a#commitcomment-151277023) for this implementation of this function